ext/transaction-process/templates/booking-request-auto-declined/booking-request-auto-declined-html.html

```diff
@@ -1,6 +1,6 @@
 {{~#*inline "format-date"~}}
 {{#with transaction.listing.availability-plan}}
-{{date d format="hh:mm a" tz=timezone}} on {{date date format="MMM d, YYYY" tz=timezone}}
+{{date date format="h:mm a" tz=timezone}} on {{date date format="MMM d, YYYY" tz=timezone}}
```

Comment:
I'm not sure if this "date date" is clear for everybody.

Reply:
Indeed, it's slightly odd. I used that as that's how our email templates have been previously. Furthermore, the `d` was a mistake, this and all other inline date formatters use param name `date`.